src/sgame/sg_cmds.cpp

```diff
@@ -381,7 +382,7 @@ void ScoreboardMessage( gentity_t *ent )
 		}
 		else
 		{
-			ping = cl->ps.ping < 999 ? cl->ps.ping : 999;
+			ping = pings[ level.sortedClients[ i ] ];
```

Comment:
Why is the check for 999 ping dropped here?

Reply:
It's already enforced in SV_CalcPings.